src/core/render/jsx/mark_dirty.ts

```diff
@@ -126,34 +129,39 @@ export function toAttrQuery(key: string): any {
  * @returns a `Promise` of all of the `HostElements` which were re-rendered.
  * @internal
  */
-export function scheduleRender(document: QDocument): Promise<HostElements> {
-  const promise = document.$qScheduledRender;
+export function scheduleRender(doc: QDocument): Promise<HostElements> {
+  const requestAnimationFrame = doc.defaultView!.requestAnimationFrame!;
```

Comment:
Is there a reason why you pulled this line up? Now it is read even if `promise` already exists.

Reply:
just to group and reduce the `const`s so there's only one, but will put it back